### DISCUSSION TITLE
Add a search button to the JSON Schema specification documents

### DISCUSSION THREAD

**Discussion by AymarN** [json-schema-org/community#759] [Specification]
Dear All,
I am writing unto you to discuss about an enhancement. Is there a way to add a search button in the specification to locate simply attributes or properties.For Instance, I am reading  the validation spec. It is convenient over the phone to use a search bar to follow the discussion regarding  "readOnly" and "writeOnly" values.
I am hoping to hear from you.
Thank You.

![1 A Search bar inside documentation](https://github.com/json-schema-org/community/assets/35435075/38167af6-6395-45f7-a9e4-d4942ae35c99)

![Validation spec 2020-12 Android](https://github.com/json-schema-org/community/assets/35435075/4cdea000-4628-42fb-9c6a-eb056cb199f5)

**Comment by gregsdennis** [comment]
Browsers (even mobile browsers) have a "find text" function built into them, and the spec document does have a table of contents.

**Reply by AymarN** [reply]
It is true that on Chrome or Firefox , there is the built -in function "find text". I understood that you would like to avoid redundancies. 
However the browser differs from the specification.

In the next release you are moving forward to your next tool chain. Whilst the outcome is to seek interactive or dynamic pages, the search button might be beneficial.

**Comment by jdesrosiers** [comment]
The spec documents are generated as IETF Internet-Draft documents. Because we're using the IETF tool chain, we don't have the ability to add enhancements like this. However, if you go to https://json-schema.org, there's a search feature that does include the spec documents. That might be helpful to search for what you're looking for.

In the next release we're abandoning the IETF tool chain and using our own, so adding a search feature is a possibility, just not quite yet and not for past releases.

**Reply by AymarN** [reply]
Thank you for your feedback. Let us hold on to the next release and discuss the tool chain results.
